core/classes/Database/DB.php

```diff
@@ -520,10 +520,10 @@ public function addColumn(string $table, string $column, string $attributes): bo
      *                        column, operator (default =), value, and glue (default AND).
      * @return array The where clause string, and parameters to bind.
      */
-    public static function makeWhere(array $clauses): array
+    private function makeWhere(array $clauses): array
```

Comment:
It should stay public it has nice use cases, And sam uses it in the new resources module he is working on

Reply:
Okay - thanks! Is the code for that one public at all? I'd like to see if we can instead improve our DB class instead of exposing its internal functions

Reply:
This is atleast one place where i see its used and looks to be a nice use case https://github.com/samerton/Nameless-Resources/blob/develop/upload/modules/Resources/classes/Resources.php#L121

Reply:
undid changes